score-api/src/main/java/io/cloudslang/score/events/FastEventBusImpl.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2014-2017 EntIT Software LLC, a Micro Focus company (L.P.)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudslang.score.events;
+
+public class FastEventBusImpl implements FastEventBus {
+
+    private ScoreEventListener eventHandler;
+
+    public void registerEventListener(ScoreEventListener eventHandler) {
+        if (this.eventHandler == null ) {
+            this.eventHandler = eventHandler;
+        } else {
+            String eventHandlerToRegister = eventHandler != null ? eventHandler.getClass().getName() : "";
```

Comment:
maybe use requireNotNull for eventHandler param in the first line of the `registerEventListener `method and then you can remove the conditional on the else branch